### PR TITLE
Docs - add table of contents per topic

### LINKED ISSUE #78: Add table of contents to documentation
**Is your feature request related to a problem? Please describe.**
Add table of contents to documentation

**Describe the solution you'd like**
In order to quickly provide an overview to anyone looking at the documentation, provide a table of contents, which:
- links to the actual section
- provides a quick overview of capabilities

Once this is done, we can publish the documentation as final (no preview) so it becomes available once the modules are published onto the PowerShell Gallery.

**Additional context**
As an example:
image

### PR DESCRIPTION
**Pull request by stijnmoreels**
Closes #78

### CODE REVIEW

File: docs/preview/features/powershell/azure-data-factory.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ layout: default
 
 # Azure Data Factory
 
+This module provides the following capabilities:
+- [Enabling a trigger of a Azure Data Factory resource](#enable-the-trigger-of-a-azure-data-factory)
```
Comment (tomkerkhove) [inline docs/preview/features/powershell/azure-data-factory.md:9]:
```suggestion
- [Enabling a trigger of a Azure Data Factory pipeline](#enable-the-trigger-of-a-azure-data-factory)
```

File: docs/preview/features/powershell/azure-data-factory.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ layout: default
...
+- [Disabling a trigger of a Azure Data Factory resource](#disable-the-trigger-of-a-azure-data-factory)
```
Comment (tomkerkhove) [inline docs/preview/features/powershell/azure-data-factory.md:10]:
```suggestion
- [Disabling a trigger of a Azure Data Factory pipeline](#disable-the-trigger-of-a-azure-data-factory)
```

File: docs/preview/features/powershell/azure-devops.md [hunk 3]
```diff
@@ -5,6 +5,9 @@ layout: default
 
 # Azure DevOps
 
+This module provides the following capabilities:
+- [Seting a variable in a DevOps pipeline](#set-devops-variable)
```
Comment (tomkerkhove) [inline docs/preview/features/powershell/azure-devops.md:9]:
```suggestion
- [Setting a variable in an Azure DevOps pipeline](#set-devops-variable)
```

File: docs/preview/features/powershell/azure-key-vault.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ layout: default
 
 # Azure Key Vault
 
+This module provides the following capabilities:
+- [Getting all access policies of a Azure Key Vault](#get-all-access-policies-of-a-azure-key-vault)
```
Comment (tomkerkhove) [inline docs/preview/features/powershell/azure-key-vault.md:9]:
```suggestion
- [Getting all access policies for an Azure Key Vault](#get-all-access-policies-of-a-azure-key-vault)
```

File: docs/preview/features/powershell/azure-storage.md [hunk 5]
```diff
@@ -5,6 +5,9 @@ layout: default
 
 # Azure Storage
 
+This module provides the following capabilities:
+- [Creating a new table in a Azure Table Storage on Azure Storage Account](#create-new-table-in-azure-table-storage-on-azure-storage-account)
```
Comment (tomkerkhove) [inline docs/preview/features/powershell/azure-storage.md:9]:
```suggestion
- [Creating a new table in an Azure Storage Account](#create-new-table-in-azure-table-storage-on-azure-storage-account)
```

File: docs/preview/features/powershell/azure-data-factory.md (hunk 1 above)
Comment (mbraekman) [reply docs/preview/features/powershell/azure-data-factory.md:9]:
```suggestion
- [Enabling a trigger of an Azure Data Factory pipeline](#enable-the-trigger-of-a-azure-data-factory)
```

File: docs/preview/features/powershell/azure-data-factory.md (hunk 2 above)
Comment (mbraekman) [reply docs/preview/features/powershell/azure-data-factory.md:10]:
```suggestion
- [Disabling a trigger of an Azure Data Factory pipeline](#disable-the-trigger-of-a-azure-data-factory)
```

File: docs/preview/features/powershell/azure-key-vault.md [hunk 6]
```diff
@@ -13,9 +17,9 @@ To have access to the following features, you have to import the module:
 PS> Import-Module -Name Arcus.Scripting.KeyVault
 ```
 
-## Get all access policies of a key vault
+## Get all access policies of a Azure Key Vault
```
Comment (mbraekman) [inline docs/preview/features/powershell/azure-key-vault.md:20]:
```suggestion
## Get all access policies of an Azure Key Vault
```

**Comment by mbraekman** [comment]
Made some small changes based on stuff that was missed during the last review.
But looks good to me.

**Pull request merged**